examples/va/nfv/sriov/edpm/nova_sriov.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# CHANGEME:
+#
+# - Change cpu-pinning-nova ConfigMap's "data['25-cpu-pinning-nova.conf']" to use CPU sets
+#   proper for your environment
+# - Change sriov-nova ConfigMap's "data['03-sriov-nova.conf']" "devname" and "physical_network"
+#   parameters to the proper SRIOV interface and network for your environment
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cpu-pinning-nova
+data:
+  25-cpu-pinning-nova.conf: |
+    [compute]
+    cpu_shared_set = 0-3,24-27 # CHANGEME
+    cpu_dedicated_set = 8-23,32-47 # CHANGEME
+    reserved_host_memory_mb = 4096
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriov-nova
+data:
+  03-sriov-nova.conf: |
+    [pci]
+    passthrough_whitelist = { "devname":"eno4", "physical_network":"sriov-phy4", "trusted":"true" } # CHANGEME
```

Comment:
"devname":"eno4", is deprecated in 17.1 downstream and it will be unsupported in 18 just an FYI so we should not use devname or passthough_whitelist in the VA
```
[PCI]
passthrough_whitelist
```
and 
```
[DEFAULT]
pci_passthrough_whitelist
```
are both replaced by 
```
[pci]
device_spec
```

i.e.
```
device_spec = {"vendor_id":"a2d6",
                   "product_id":"15b3",
                   "address": "0000:82:00.0",
                   "physical_network":"physnet1",
               }
```